TPI/app/src/main/java/biz/stratadigm/tpi/ui/fragment/VenuesFragment.java

```diff
@@ -0,0 +1,192 @@
+package biz.stratadigm.tpi.ui.fragment;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.error.VolleyError;
+import com.android.volley.request.StringRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import biz.stratadigm.tpi.App;
+import biz.stratadigm.tpi.R;
+import biz.stratadigm.tpi.di.component.DaggerVenuesComponent;
+import biz.stratadigm.tpi.entity.dto.VenueDTO;
+import biz.stratadigm.tpi.entity.vo.VenueVO;
+import biz.stratadigm.tpi.tools.Constant;
+import biz.stratadigm.tpi.ui.adapter.VenueAdapter;
+import biz.stratadigm.tpi.ui.view.VenuesView;
+import biz.stratadigm.tpi.di.module.VenuesModule;
+import biz.stratadigm.tpi.presenter.VenuesPresenter;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import nucleus.factory.PresenterFactory;
+
+//public class VenuesFragment extends Fragment {
+public class VenuesFragment extends BaseFragment<VenuesPresenter> implements VenuesView {
+
+    private static final String TAG = "TPI";
+
+    @BindView(R.id.venueList)
+    RecyclerView mList;
+
+    @Inject
+    VenuesPresenter venuesPresenter;
+    {
+        DaggerVenuesComponent.builder()
+                .appComponent(App.getAppComponent())
+                .venuesModule(new VenuesModule())
+                .build()
+                .inject(this);
+    }
+   
+    private RecyclerView.LayoutManager mLayoutMAnager;
+    //private ArrayList<VenueDTO> mListVenue = new ArrayList<>();
+    private VenueAdapter mVenueAdapter;
+    private SharedPreferences sharedPreferences;
+    private int offset = 0;
+    private TextView more, less;
+
+    //public static VenuesFragment newInstance() {
+    //    return new VenuesFragment();
+    //}
+
+    @Override
+    public PresenterFactory<VenuesPresenter> getPresenterFactory() {
+        return () -> venuesPresenter;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.venue_list_fragment, container, false);
```

Comment:
It would be nice, if you renamed venue_list_fragment to fragment_venue_list. Such prefixes implicitly group layout files by type(activity_, fragment_, item_, block_, header_, footer_), so it makes easier to find a layout file